services/QuillLMS/app/controllers/teachers/classroom_manager_controller.rb

```diff
@@ -244,6 +244,12 @@ def unset_view_demo
     redirect_to '/profile'
   end
 
+  # rubocop:disable Naming/PredicateName
+  def is_demo
```

Comment:
In general, the Naming/PredicateName is a pretty solid cop.  There's some discussion [here](https://github.com/rubocop/rubocop/issues/4479) concerning it's application to Rails controllers, but I would agree with their conclusion.  In this case,  `is_demo` conveys boolean semantics but all that's being passed is a hash; the actual boolean check is in the [component](https://github.com/empirical-org/Empirical-Core/pull/9468/files#diff-ef59189626d7a911f9682a2800e3f0e94968aea1b92a8dbbedc38e4820bba37bR147) itself.   What about `def current_user_demo_id` instead?

Reply:
What if we just called this action `demo_id` or something?